### PR TITLE
Make container id mandatory

### PR DESCRIPTION
**Pull request by jodh-intel**

### CODE REVIEW

**Comment by jodh-intel** [comment]
Blocked on https://github.com/containers/virtcontainers/issues/414.

**Comment by jodh-intel** [comment]
Now blocked on https://github.com/containers/virtcontainers/pull/415.

**Comment by grahamwhaley** [comment]
jodh - indirect question - did you add the DNM label before you submitted, or just after? I'm wondering if our Jenkins CI honors that and does not try a a build etc. (as I see you have DNM here with failed CIs). /cc @sboeuf @chavafg wondering if I need to open a Jenkins ticket or not to add filtering?

File: src/shim.c [hunk 1]
```diff
@@ -704,7 +704,7 @@ handle_proxy_notification(struct cc_shim *shim, struct frame *fr)
 				"to proxy at %s\n", shim->proxy_address);
 		}
 
-		shim_debug("Exit status for container: %d\n", code);
+		shim_debug("Exit status for container %s: %d\n", code);
```
Comment (grahamwhaley) [inline src/shim.c]: Have you missed the `shim->container_id` new/extra argument here?

**Review by grahamwhaley — CHANGES_REQUESTED** [review]
lgtm
One minor query.

File: src/shim.c (hunk 1 above)
Comment (jodh-intel) [reply src/shim.c]: Good catch! I'll update...

Comment (jodh-intel) [reply src/shim.c]: Fixed.

**Review by grahamwhaley — APPROVED** [review]
lgtm

**Comment by jodh-intel** [comment]
https://github.com/containers/virtcontainers/pull/415 has now landed as has the vc revendor required for this change (https://github.com/clearcontainers/runtime/pull/722), so removing dnm label...

**Review by amshinde — APPROVED** [review]
lgtm

**Review by sboeuf — APPROVED** [review]

**Comment by sboeuf** [comment]
LGTM

**Pull request merged**